packages/forms/src/model.ts

```diff
@@ -1374,15 +1374,15 @@ export class FormControl extends AbstractControl {
    *
    * @param fn The method that is called when the value changes
    */
-  registerOnChange(fn: Function): void {
+  registerOnChange(fn: (value?: any, emitModelEvent?: boolean) => void): void {
```

Comment:
Can we use `unknown` here instead? The provided function may return something, but we are not interested in the result (we don't use the return value anywhere), so `unknown` looks like a good fit.
```suggestion
  registerOnChange(fn: (value?: any, emitModelEvent?: boolean) => unknown): void {
```

Reply:
I think that is actually really misleading -- the callback is fired from our code, so it's not possible that the return value does anything useful. I worry that people would be able to call this method expecting their return value to be used, and that's a misleading API. WDYT?

Reply:
@dylhunn yeah, that makes sense, let's keep `void` 👍  As you mentioned, this sends a stronger signal to an API user that the callback function invocation result is unused.